src/SessionMessages/SessionMessage/MessageFiles.tsx

```diff
@@ -12,23 +12,63 @@ interface MessageFilesProps extends PropsWithChildren {
   files: ConversationFile[];
 }
 
+
 export const MessageFiles: FC<MessageFilesProps> = ({ files, children }) => {
   const { theme } = useContext(ChatContext);
   const Comp = children ? Slot : MessageFile;
+  const [expanded, setExpanded] = useState<boolean>(false);
 
   if (!files || files.length === 0) {
     return null;
   }
 
+  // Filter image files
+  const imageFiles = files.filter(file => file.type.startsWith('image/'));
```

Comment:
These 2 itterations could be combined into one using a grouping or something like:

```
const { imageFiles, otherFiles }  = files.reduce((acc, cur) => {}, { imageFiles: [], otherFiles: [] })
```